atomic_reactor/util.py

```diff
@@ -1671,41 +1680,41 @@ def base_image_key(self):
             raise KeyError('base_image')
 
     @property
-    def base_from_scratch(self):
+    def base_from_scratch(self) -> bool:
         """
         base image is FROM scratch
         :return: boolean
         """
         return self._base_from_scratch
 
     @property
-    def custom_base_image(self):
+    def custom_base_image(self) -> bool:
         """
         base image is custom (FROM koji/image-build)
         :return: boolean
         """
         return self._custom_base_image
 
     @property
-    def custom_parent_image(self):
+    def custom_parent_image(self) -> bool:
         """
         any parent image is custom (FROM koji/image-build)
         :return: boolean
         """
         return self._custom_parent_image
 
     @property
-    def source_registry(self):
+    def source_registry(self) -> Optional[str]:
         return self._source_registry
 
     @property
-    def organization(self):
+    def organization(self) -> Optional[str]:
         return self._organization
 
-    def __len__(self):
+    def __len__(self) -> int:
         return len(self._pullable_parents)
 
-    def __setitem__(self, key, item):
+    def __setitem__(self, key: Union[str, ImageName], item: str):
```

Comment:
Technically `item` can be an ImageName too, not sure if it's used that way though (`ImageName.parse()` works on ImageName objects too)